ReactWindows/ReactNative.Shared/UIManager/UIManagerModule.cs

```diff
@@ -498,18 +511,24 @@ public override void OnReactInstanceDispose()
 
         #endregion
 
-        #region Dimensions
-
+#region Dimensions
+#if WINDOWS_UWP
         private void OnBoundsChanged(ApplicationView sender, object args)
         {
             Context.GetJavaScriptModule<RCTDeviceEventEmitter>()
                 .emit("didUpdateDimensions", GetDimensions());
         }
+#endif
 
         private static IDictionary<string, object> GetDimensions()
         {
+#if WINDOWS_UWP
             var bounds = ApplicationView.GetForCurrentView().VisibleBounds;
             var scale = DisplayInformation.GetForCurrentView().RawPixelsPerViewPixel;
+#else
+            var bounds = Application.Current.Windows.OfType<Window>().FirstOrDefault(w => w.IsActive);
+            var scale = new Double();
```

Comment:
Yeah, totally agree. Part of the reason for opening this PR was so we could discuss this problem.